story/tests.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.utils import timezone
+from story.models import Story
+from house.models import House, Country, City
+
+TEMP_DATE = timezone.now()
+TEMP_STORY = "This is my story"
+
+HOUSE_NAME = "House 1"
+HOUSE_PUBLIC = True
+COUNTRY = "Colorado"
+CITY = "South Park"
+HOUSE_PARENT_PROFESSION_1 = "Teacher"
+HOUSE_PARENT_PROFESSION_2 = "Student"
+country = Country(name=COUNTRY)
+city = City(name=CITY, country=country)
+house = House(name=HOUSE_NAME, public=HOUSE_PUBLIC, country=country, city=city, income=1, children=1)
+
+
+@pytest.mark.django_db()
+def test_new_item():
+    country.save()
+    city.save()
+    house.save()
+    item = Story(story_date=TEMP_DATE, story_text=TEMP_STORY, house=house)
+    item.save()
+
+    assert item.story_text == TEMP_STORY
+    assert item.story_date == TEMP_DATE
+    assert item.house.house_id == house.house_id
+
+
+@pytest.fixture
+def story0():
+    return Story(story_date=TEMP_DATE, story_text=TEMP_STORY, house=house)
```

Comment:
you should be persistent throughout the project. 
You have decided as far as I know to use CRUD methods (https://github.com/redhat-beyond/Xpense/blob/e00f90465ec146a84a3acd052120c644001f83b6/house/models.py#L60)